validation/copyStaticToDist.sh

```diff
@@ -1,4 +1,3 @@
-# Copies files from src/public/ and src/lib/output/ to dist/
-cp -R ./src/public/*.html ./dist/public
-cp -R ./src/public/styles ./dist/public
-cp -R ./src/lib/output ./dist/lib
+# Copies files from src/static/ to dist/
+cp -R ./src/static/*.html ./dist/static
+cp -R ./src/static/styles ./dist/static
```

Comment:
but not `/static/scripts`?

Reply:
They are actually brought into `dist/` by typescript's `tsc`. I will add in a line to copy `static/scripts` when I have a dependency on some external `.js` library (probably the next PR).